core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -435,7 +435,7 @@ package object config {
         "a property key or value, the value is redacted from the environment UI and various logs " +
         "like YARN and event logs.")
       .regexConf
-      .createWithDefault("(?i)secret|password".r)
+      .createWithDefault("(?i)secret|password|token".r)
```

Comment:
To keep it targeted, match oauthToken?

Reply:
OK. This might catch tokens we don't want to redact, but I can't think of any tokens we don't want to redact, either.